Amplify/Categories/API/ClientBehavior/AmplifyAPICategory+GraphQLBehavior.swift

```diff
@@ -67,4 +68,8 @@ extension AmplifyAPICategory: APICategoryGraphQLBehavior {
             plugin.subscribe(toAnyModelType: modelType, subscriptionType: subscriptionType, listener: listener)
     }
 
+    @available(iOS 13.0, *)
+    public func reachability(apiName: String, allowsCellularAccess: Bool) -> AnyPublisher<ReachabilityUpdate, Never>? {
```

Comment:
should this be here in Model based GraphQL operations or something else like APICategoryReachabilityBehavior?

Reply:
Updated.